functions/openhabian.bash

```diff
@@ -241,3 +241,38 @@ config_ipv6() {
     if cond_redirect sysctl --load; then echo "OK"; else echo "FAILED"; return 1; fi
   fi
 }
+
+
+## Create UNIX user and group for openHABian administration purposes
+##
+##    create_user_and_group()
+##
+create_user_and_group() {
+
+# TODO: add calling this routine in openhabian-setup.bash (or elsewhere ?)
+
+# IF
+# (1) the string/username that the end user entered as "username=" in openhabian.conf is *empty* OR
```

Comment:
adminusername= not username=

Reply:
err, yes. I wrote that comment before I renamed username to adminusername

Reply:
corrected